plugin/autopath/setup.go

```diff
@@ -11,12 +11,14 @@ import (
 	"github.com/miekg/dns"
 )
 
-func init() { plugin.Register("autopath", setup) }
+const pluginName = "autopath"
```

Comment:
I'm still not a fan of this PR and intent to close it.

Technically we don't need this const because we can use `<plugin>.Name()` method

Reply:
Let me see about making use of that. I think it's a bit expensive to call a string method, where a simple const at compile time will do.